modules/tensor_mechanics/src/materials/ADViscoplasticityStressUpdate.C

```diff
@@ -329,3 +332,27 @@ ADViscoplasticityStressUpdate<compute_stage>::computeInelasticStrainIncrement(
   inelastic_strain_increment =
       _dt * dpsi_dgauge * computeDGaugeDSigma(gauge_stress, equiv_stress, dev_stress, stress);
 }
+
+template <ComputeStage compute_stage>
+void
+ADViscoplasticityStressUpdate<compute_stage>::outputIterationSummary(
+    std::stringstream * iter_output, const unsigned int total_it)
+{
+  if (iter_output)
+  {
+    *iter_output << "At element " << _current_elem->id() << " _qp=" << _qp << " Coordinates "
```

Comment:
This is something we should probably add to the other classes that use the return mapping solution as well.

Reply:
As in, put this into `ADSingleVariableReturnMappingSolution<compute_stage>::outputIterationSummary`?

Reply:
`ADSingleVariableReturnMappingSolution` doesn't know about `_qp`, it'd have to be in `ADRadialReturnStressUpdate`. Doable, but I'll leave it to someone else.

Reply:
Yeah, exactly.  I think we would need to put this in every class that uses `ADSingleVariableReturnMappingSolution` because that's the level where the elem and qp are available.  I'm not sure if there's a cleaner way to do it.  Don't worry about it.